orga/app/components/auth/login-form.js

```diff
@@ -51,6 +51,8 @@ export default class LoginForm extends Component {
           }
         });
       }
+    } else {
+      this.isLoading = false;
```

Comment:
**suggestion**
doublon avec ce qui se passe dans this._authenticate ?

Reply:
corrigé !